### PR TITLE
translation page added

### PR DESCRIPTION
**Pull request by Labhk**

### CODE REVIEW

**Comment by GauravBurande** [comment]
I think it's not working, the deploy preview is failed. Fix the error

**Review by GauravBurande — COMMENTED** [review]
Have you used api key? I see only the dummy text there

**Comment by GauravBurande** [comment]
Use env variable to store the api key

File: src/components/Translations.jsx [hunk 1]
```diff
@@ -1,13 +1,107 @@
-import React from 'react'
-import WorkingOn from './utils/WorkingOn'
+import React,{useEffect,useState} from 'react'
+import WorkingOn from './utils/WorkingOn';
```
Comment (GauravBurande) [inline src/components/Translations.jsx]: remove this line as we're not using the component anymore

File: src/components/Translations.jsx [hunk 2]
```diff
@@ -1,13 +1,107 @@
...
+  const [output, setOutput] = useState('');
+    const [array, setArray]  = useState([])
+  
+    const defaultText = ["Hello","How are you", "My name is George", " Thank you","Where is this place"]
+
+
+
+    const defaultTranslate = () => {
+        setArray([])
+        defaultText.map((text) => {
```
Comment (GauravBurande) [inline src/components/Translations.jsx:21]: this line is causing error Array.prototype.map() expects a return value from arrow function, please have a look at ur code again

Comment (GauravBurande) [reply src/components/Translations.jsx:21]: have a return value for ur map function, maybe it'll solve the error

File: src/components/Translations.jsx [hunk 3]
```diff
@@ -1,13 +1,107 @@
...
+            <option key={opt.code} value={opt.code}>
+              {opt.name}
+            </option>
+          ))}
+        </select>  
+      
+      <button className='border border-solid rounded-lg ' onClick={defaultTranslate}>Translate</button>
+      <h3>Hello    ----   {array[0]}</h3>
+      <h3>How are you? ----  {array[1]}</h3>
+      <h3>My name is George -----  {array[2]}</h3>
```
Comment (GauravBurande) [inline src/components/Translations.jsx]: enhance the UI of this page, maybe use flex box at some place and table if you can.

**Review by GauravBurande — CHANGES_REQUESTED** [review]
I've found where is the error, please try to fix it and enhance the UI

File: src/components/Translations.jsx (hunk 1 above)
Comment (GauravBurande) [reply src/components/Translations.jsx]: also use prettier extension for code formatting

File: src/components/Translations.jsx (hunk 2 above)
Comment (Labhk) [reply src/components/Translations.jsx:21]: ok

File: src/components/Translations.jsx (hunk 3 above)
Comment (Labhk) [reply src/components/Translations.jsx]: ok

**Comment by Labhk** [comment]
I work on UI and the suffling list then u can merge

**Review by GauravBurande — APPROVED** [review]

**Pull request merged**